### PR TITLE
Book detail

### PR DESCRIPTION
**Pull request by jeymarsonwolox**
**Summary**

**Book detail**

Screenshot

iPhone 6
ios6

iPhone SE
iosse

iPhone 8
ios8

iPhone X
iosx

Android 4.4
android4 4

Android 6
android6

Android 7
android7

Android 8
android8

**Trello card**

https://trello.com/c/tQ1dB5RE

### CODE REVIEW

File: todolist/app/components/Input/index.js [hunk 1]
```diff
@@ -29,7 +30,7 @@ class Input extends Component {
   render() {
     return <TextInput
       placeholder="Enter an item!"
-      placeholderTextColor="grey"
+      placeholderTextColor={GREY}
```
Comment (julibel) [inline todolist/app/components/Input/index.js]: constants for color should be lowerCase.. just conventions 🤷‍♀️

File: todolist/app/screens/BookDetail/index.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import React, { Component } from 'react';
+
+import Detail from './layout';
+
+class BooKDetail extends Component {
+  static navigationOptions = ({ navigation }) => {
+    return {
+      title: navigation.state.params.item.title
```
Comment (julibel) [inline todolist/app/screens/BookDetail/index.js]: use getParam here

File: todolist/app/screens/ListBook/index.js [hunk 3]
```diff
@@ -2,17 +2,15 @@ import React from 'react';
...
 
 import styles from './styles';
-import Title from '../../components/Title';
 import List from '../../components/List';
 import Book from '../../components/Book';
 
 import books from '../../constants/listBoook';
 
-function App() {
+function App({ navigation }) {
```
Comment (julibel) [inline todolist/app/screens/ListBook/index.js]: be careful with naming

File: todolist/app/screens/Todo/index.js [hunk 4]
```diff
@@ -13,7 +12,6 @@ import Checkbox from '../../components/Checkbox';
 function App({ items }) {
```
Comment (julibel) [inline todolist/app/screens/Todo/index.js]: idem

**Review by julibel — CHANGES_REQUESTED** [review]
Nice!

File: todolist/android/gradle.properties [hunk 5]
```diff
@@ -16,3 +16,7 @@
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+TODOLIST_RELEASE_STORE_FILE=todolist-key.keystore
```
Comment (julibel) [inline todolist/android/gradle.properties]: okey.. first, you should't commit this to github. This are used to sign the apk, so is kind of an important key. We try to never add keys or similar to git. Also.. this credentials are in the wrong place, we should use them in a system file not in the project. We will check this in a call.. just delete this line and we are good

File: todolist/app/config/ReactotronConfig.js [hunk 6]
```diff
@@ -1,8 +1,9 @@
 import Reactotron from 'reactotron-react-native';
+import { reactotronRedux } from 'reactotron-redux';
 
-export default function config() {
-  Reactotron
-  .configure()
-  .useReactNative()
+const reactotron = Reactotron
+  .configure({ name: 'React Native Demo' })
+  .use(reactotronRedux()) //  <- here i am!
```
Comment (julibel) [inline todolist/app/config/ReactotronConfig.js:6]: `hello Reactotron!`

File: todolist/app/screens/BookDetail/index.js [hunk 7]
```diff
@@ -0,0 +1,20 @@
+import React, { Component } from 'react';
+
+import Detail from './layout';
+
+class BooKDetail extends Component {
+  static navigationOptions = ({ navigation }) => {
```
Comment (julibel) [inline todolist/app/screens/BookDetail/index.js:6]: what's this for?

File: todolist/app/screens/BookDetail/index.js [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+    const item = navigation.getParam('item',{});
+    return {
+      title: item.title
+    }
+  };
+  render() {
+    const item = this.props.navigation.getParam('item',{});
```
Comment (julibel) [inline todolist/app/screens/BookDetail/index.js]: `this.props.navigation.getParam('item');` i don't think that you need the `{}` part

File: todolist/app/screens/BookDetail/layout.js [hunk 9]
```diff
@@ -0,0 +1,34 @@
...
+import PropTypes from 'prop-types';
+import { ScrollView, View, Image } from 'react-native';
+
+import styles from './styles';
+import Info from '../../components/Info';
+
+function Detail({ item }) {
+  return (
+    <ScrollView style={styles.container}>
+      {item.image_url && <Image   style={styles.img} source={{uri: item.image_url}} />}
```
Comment (julibel) [inline todolist/app/screens/BookDetail/layout.js]: you have some more spaces between Image and style

File: todolist/app/screens/ListBook/index.js [hunk 10]
```diff
@@ -2,17 +2,15 @@ import React from 'react';
...
 
 import books from '../../constants/listBoook';
 
-function BookList() {
+function BookList({ navigation }) {
   return (
     <View style={styles.container}>
-      <Title value={'Book List'}/>
-      <List ComponentChild={Book} items={books}/>
+      <List ComponentChild={Book} items={books} navigate={navigation.navigate} />
```
Comment (julibel) [inline todolist/app/screens/ListBook/index.js:13]: mmm i'm not convinced.. you should not need to pass navigation to the List. You can use NavigationActions directly from react-navigation:
`import { NavigationActions } from 'react-navigation';`
And then you can use it the same way that you are using navigate prop:
`dispatch(NavigationActions.navigate({ routeName: Routes.RecoverPassword }));`

**Review by julibel — CHANGES_REQUESTED** [review]

File: todolist/app/screens/BookDetail/index.js (hunk 7 above)
Comment (jeymarsonwolox) [reply todolist/app/screens/BookDetail/index.js:6]: get the title of the book and put it in the header

**Review by julibel — APPROVED** [review]

**Pull request merged**